src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -669,9 +670,9 @@ function transpiration_per_ground_area(
     transpiration::PrescribedTranspiration{FT},
     _,
     _,
-    t::FT,
-)::FT where {FT}
-    return transpiration.T(t) # (m/s)
+    t,
+)::FT where {FT <: AbstractFloat}
+    return FT(transpiration.T(t)) # (m/s)
```

Comment:
I think we are also OK here without saying FT<:AbstractFloat (and just "FT"), because of https://github.com/CliMA/ClimaLSM.jl/blob/ad31773009c9309d5e45b083603873a8914be286/src/standalone/Vegetation/PlantHydraulics.jl#L54